tests/robot-tests/tests/libs/tables-common.robot

```diff
@@ -58,7 +59,8 @@ user gets table row with heading
     [Return]    ${elem}
 
 user gets table row
-    [Arguments]    ${row_cell_text}    ${parent}=css:table
+    [Arguments]    ${row_cell_text}    ${parent}=css:table    ${wait}=%{WAIT_SMALL}
```

Comment:
As above comment - does this need a wait? Similarly, we should be able to use some other keyword to wait for the table (or something in it) before checking table rows.